src/processor/reconnection-graph.service.ts

```diff
@@ -130,13 +130,12 @@ export class ReconnectionGraphService {
       }
     } catch (err: any) {
       this.logger.error(`Error updating graph for user ${dsnpUserStr}, provider ${providerStr}: ${(err as Error).stack}`);
-      if (updateConnections) {
-        /// if updateConnections is true, we want to queue a graph update job and pause the queue
-        this.graphUpdateQueue.add('graphUpdate', dataNT, { jobId: jobIdNT });
-      }
-      // if we have unknwon error, capacity error we want to pause the queue
-      if (err instanceof errors.UnknownError || err instanceof errors.CapacityLowError || err instanceof errors.GetUserGraphError) {
-        this.graphUpdateQueue.pause();
+      if (err instanceof errors.UnknownError || err instanceof errors.GetUserGraphError) {
+        if (updateConnections) {
+          /// if updateConnections is true, we want to queue a graph update job and pause the queue
+          this.graphUpdateQueue.add('graphUpdate', dataNT, { jobId: jobIdNT });
+        }
+        this.eventEmitter.emitAsync('error.graph', err);
```

Comment:
emit error as event when errors are arising from some unknown source or bad configuration, so event will async'ly pause the queue, since these errors would mean something wrong with env/config

Reply:
@wilwade  so here it emits an event which goes and pause the queue for these errors and simultaneous fails the job by throwing back the error